src/dualscreen.tsx

```diff
@@ -72,15 +72,15 @@ export class DualScreen extends KalturaPlayer.core.BasePlugin implements IEngine
     return this._readyPromise;
   }
 
-  getUIComponents() {
-    return [
-      {
-        presets: PRESETS,
-        container: ReservedPresetAreas.BottomBarRightControls,
-        get: KalturaPlayer.ui.components.Remove,
-        replaceComponent: KalturaPlayer.ui.components.Settings.displayName
-      }
-    ];
+  private _removeSettingsComponent() {
+    const removeSettings =
+        {
+          presets: PRESETS,
+          container: ReservedPresetAreas.BottomBarRightControls,
+          get: KalturaPlayer.ui.components.Remove,
+          replaceComponent: KalturaPlayer.ui.components.Settings.displayName
+        }
+    this._player.ui.addComponent(removeSettings);
```

Comment:
Please save the return function of addComponent lets say call it undoRemoveSettings or something and call it in reset like we discussed so when someone changes the media to some non dual media the button will return.
Check this scenario of course